### ISSUE TITLE
Run instrumentation tests on Firebase.

### ISSUE THREAD

**Issue by artem-zinnatullin** [artem-zinnatullin/qualitymatters#220]
```console
0.47s$ echo no | android create avd --force -n test -t android-18 --abi armeabi-v7a
Error: Target id is not valid. Use 'android list targets' to get the target ids.
The command "echo no | android create avd --force -n test -t android-18 --abi armeabi-v7a" failed and exited with 1 during .
```

I'm tired of Travis flakiness and now android sdk update broke the repo.

**Comment by knightcube** [comment]
How do i run instrumentation tests on firebase?

**Comment by artem-zinnatullin** [comment]
https://firebase.google.com/docs/test-lab/

Not sure what do you ask here, you mean how to set Firebase Test Lab for this particular project or just in general?

**Comment by knightcube** [comment]
yeah sorry to be not specific. I am a newbie in github and i want to understand how to solve issues of a particular repository. For instance) In this particular issue should i test the app using Firebase test labs? If yes then what else should i do to make a significant contribution to this repository?

**Comment by artem-zinnatullin** [comment]
You can start with something simple, like updating dependencies, then just pick an [issue](https://github.com/artem-zinnatullin/qualitymatters/issues) and comment with possible solution so we could agree on implementation before the PR :)

**Comment by knightcube** [comment]
How can i update dependencies?...Is there any link or documentation that can help me to do that?

**Comment by artem-zinnatullin** [comment]
Well, you probably need to start with [Gradle Dependency Management Basics](https://docs.gradle.org/current/userguide/artifact_dependencies_tutorial.html) documentation

**Comment by mcassiano** [comment]
Do you need help? I've done this before for a small app, but using CircleCI. I think there are a few differences in the configuration file for Travis, but it shouldn't be that hard.

**Comment by artem-zinnatullin** [comment]
Yep, I've done it but help would be appreciated 👍  (i don't have much time to do it)

The only problem is API key, we can set it up so it would work only on my PRs without leaking key but I'm not sure it adds value then…
